### DISCUSSION TITLE
The coordinates of the characters confuse me

### DISCUSSION THREAD

**Discussion by LooEv** [jsvine/pdfplumber#1015] [Ask for help with specific PDFs]
PDF: https://www.ecolab.com/-/media/Widen/Sustainability/Corporate-Sustainability-Report/2022/2022-Ecolab-Corporate-Responsibility-Report_pdf.pdf
I am processing the content of page 91 in this PDF file. The y0 and y1 coordinates of the phrases "ESG performance data" and "This appendix summarizes key environmental," are confusing me. From the PDF page, they appear to be quite far apart. However, when I use pdfplumber to process them, I found that the y0 for "ESG" is 522.5904, and the y1 for "This" is 527.027. It seems there is an overlap between them, which I find puzzling. Perhaps I have a mistaken understanding of the definitions of y0 and y1. Please correct me if I'm wrong. Thank you.

![image](https://github.com/jsvine/pdfplumber/assets/15908616/7676e68f-2791-4b10-bbe4-1d30abd752fa)

**Comment by jsvine** [comment]
You'll probably want to ignore `y0` and `y1`, which are given in the PDF's original coordinate space (in which vertical positions are measured from the bottom of the page, rather than the top). Instead, you'll likely find more useful `top` (the distance from the top of the page to the top of the object) and `bottom` (the distance from the top of the page to the bottom of the object).

**Reply by LooEv** [reply]
Thank you very much for your reply.
